release-master-to-test-environment.yml

```diff
@@ -0,0 +1,57 @@
+# Add steps that build, run tests, deploy, and more:
```

Comment:
Do we need separate pipeline to make the test deploy? Or is there a way to extend the azure-pipeline.yml?

Reply:
Or maybe we can group them somehow? by name? by using folder?

Reply:
We discussed this and decided we'll put those pipeline files into a separate folder:

- [ ] Put azure pipeline files to a separate folder